src/ui-kit/components/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,29 @@
+import { memo } from 'react';
+import { AvatarText, AvatarWrapper } from './Avatar.styled';
+
+function Avatar({
+  AvatarComponent = 'AB',
+  avatarWidth = '80px',
+  avatarHight = '80px',
```

Comment:
You do not need to pass all this, you have only 4 options, just make a size that can be sm, md, lg ...